components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.commons/src/main/java/org/wso2/carbon/cluster/coordinator/commons/configs/CoordinationPropertyNames.java

```diff
@@ -46,6 +46,11 @@
     public static final String ZOOKEEPER_CONNECTION_STRING = "connectionString";
 
     /**
+     * RDBMS coordination DataSource property name.
+     */
+    public static final String DATASOURCE_NAME = "dataSource";
```

Comment:
Shall we use a meaningful name rather than `"datasource"`. How about something like `"WSO2_COORDINATION_DB"`?